src/shared/model/CancerHotspots.ts

```diff
@@ -1,23 +1,5 @@
-import {HotspotMutation} from "shared/api/generated/CancerHotspotsAPI";
-import HotspotSet from "shared/lib/HotspotSet";
+import {AggregatedHotspots} from "shared/api/generated/GenomeNexusAPIInternal";
 
 export interface IHotspotIndex {
-    [gene:string]: {
-        [hotspotType:string]: IHotspotLookup
-    }
-}
-
-export interface IHotspotLookup {
-    hotspotMutations: HotspotMutation[],
-    hotspotSet: HotspotSet;
-}
-
-export interface IHotspotData {
-    single: IHotspotIndex,
-    clustered: IHotspotIndex
-}
-
-export interface ICancerHotspotData {
-    single: HotspotMutation[],
-    clustered: HotspotMutation[]
+    [genomicLocation: string]: AggregatedHotspots;
```

Comment:
Now we are mostly relying on the data structure provided by Genome Nexus, thus the frontend cancer hotspots model is now a lot simpler.